parallel/parallel.go

```diff
@@ -3,31 +3,16 @@ package parallel
 import (
 	"runtime"
 	"sync"
-	"syscall"
 )
 
-const minNumWorkers = 2
-const minOpenFilesLimit = 1000
-
-var global *Manager
+const (
+	minNumWorkers     = 2
+	minOpenFilesLimit = 1000
```

Comment:
we can also move `minOpenFilesLimit` into global.go since it is used by `adjustOpenFilesLimit()` func

Reply:
done.